xml/article_setup.xml

```diff
@@ -146,32 +146,32 @@
 <!-- shield cpu -->
 <screen><command>cset</command> shield --cpu=3</screen>
    <para>
-    This will shield CPU3. On a machine with 4 cores CPU0-CPU2 are unshielded.
+    This will shield CPU3. On a machine with 4 cores, CPU0-CPU2 are unshielded.
    </para>
   </sect2>
 
   <sect2 xml:id="sec-slert-quick-settingup-cpushield-multi">
    <title>Setting up CPU shields for multiple CPUs</title>
    <para>
-    If you need to shield more than one CPUs, the argument of the
-    <option>--cpu</option> option accepts comma separated lists of CPUs
+    If you need to shield more than one CPU, the argument of the
+    <option>--cpu</option> option accepts comma-separated lists of CPUs,
     including range specifications:
    </para>
 <screen><command>cset</command> shield --cpu=1,3,5-7</screen>
    <para>
-    On a machine with 8 cores, this command will shield CPU1, CPU3, CPU5, CPU6,
-    and CPU7. The CPUs CPU0, CPU2 and CPU4 will remain unshielded.
+    On a machine with eight cores, this command will shield CPU1, CPU3, CPU5,
+    CPU6, and CPU7. CPUs CPU0, CPU2 and CPU4 will remain unshielded.
    </para>
    <para>
-    Already existing CPU shields can be extended by the same command. For
-    example, to add CPU4 to the mentioned CPU set, use this command:
+    Existing CPU shields can be extended by the same command. For example, to
+    add CPU4 to the CPU set described above, use this command:
    </para>
 <screen><command>cset</command> shield --cpu=1,3-7</screen>
    <para>
-    CPU1, CPU3, CPU5 to CPU6 were already shielded and only CPU4 will
+    CPU1, CPU3, and CPU5 to CPU6 were already shielded and only CPU4 will
     additionally be shielded. Technically, the command is updating the current
-    CPU shield schema. To reduce the number of shielded CPUs and to unshield
-    CPU1, for example, use the following command:
+    CPU shield schema. To reduce the number of shielded CPUs, for example, to
+    unshield CPU1, use the following command:
```

Comment:
This all sounds like it could be easier:
* "technically" is imo filler, and the entire "Technically, ..." sentence does not really fit into the paragraph
* "CPU shield schema" sounds off -- "CPU shielding schema" would sound more natural to me
* the last sentence is too complicated for what it is trying to say, and could probably be split

Reply:
Good point. Rephrased.